fakestorage/upload.go

```diff
@@ -380,7 +380,11 @@ func (s *Server) uploadFileContent(r *http.Request) jsonResponse {
 			commit = parsed.KnownTotal && (parsed.End+1 >= parsed.Total)
 		} else {
 			// End of a streaming request
-			responseHeader.Set("Range", fmt.Sprintf("bytes=0-%d", len(obj.Content)))
+			contentLength := 0
+			if obj.Content != nil {
+				contentLength = len(obj.Content)
+			}
+			responseHeader.Set("Range", fmt.Sprintf("bytes=0-%d", contentLength))
```

Comment:
The new code is doing the same thing. `len` returns 0 for a nil slice: https://play.golang.org/p/CArFxLf4d4v

Reply:
Thanks for the tip, I'm not yet that fluent in Go.